tools/crm_mon.c

```diff
@@ -819,7 +820,7 @@ main(int argc, char **argv)
 
         do {
             if (!one_shot) {
-                print_as("Attempting connection to the cluster...\n");
+                print_as("Waiting until cluster is available on this node ...\n");
```

Comment:
FYI I noticed that if crm_mon is run immediately after starting the cluster, we get the new "critical - lost connection to daemons" messages as well. Maybe we could suppress those if we've never connected yet?

Reply:
That might have to do with this fenced-startup hickup I had noticed when I was making watchdog-fencing configurable per node.
Who should memorize that crm_mon had already connected once? Actually it would have to be something private to crm_mon as we would otherwise create a chicken & egg problem (you need to be connected to know that you shouldn't moan because you couldn't connect ...).
I've also seen that we are getting this message here (new or old alike "Waiting until cluster...") when we call crm_mon with '-w' which is as well not nice. But that should be easy to avoid.

Reply:
That wasn't '-w' but '--as-html' that doesn't set one_shot and thus prints this message that makes it look as if it was still waiting and not already repeatedly writing the html-file - for my understanding even more misleading with the new wording ...

Reply:
I wasn't able to reproduce the "critical - lost connection to daemons" message with a cmdline like
systemctl start pacemaker && sleep 5 && crm_mon -m3 -N
(with various sleeps of course)
Seems like an unfortunate race ...